demos/continuous_batching/rag/README.md

```diff
@@ -4,7 +4,7 @@
 ## Creating models repository for all the endpoints
 
 ```console
-git clone https://github.com/openvinotoolkit/model_server
+git clone -b releases/2025/0 https://github.com/openvinotoolkit/model_server
```

Comment:
Please change it to flow as above (current version may not work with automatic tests):
curl https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/releases/2025/0/demos/common/export_models/export_model.py -o export_model.py
pip3 install -r https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/releases/2025/0/demos/common/export_models/requirements.txt